utils/contributors.js

```diff
@@ -31,7 +31,7 @@ const CONTRIBUTERS = [
                     <lord-icon src="https://cdn.lordicon.com/jpdtnwas.json" trigger="loop" colors="primary:#121331,secondary:#e88c30" style="width:25px;height:25px"></lord-icon>
             </div>
         `,
-        github: "https://github.com/ankur12-1610",
+        github: "https://github.com/myst-adiTya07",
```

Comment:
You have to create a new card with your details and not edit the details from existing ones.